core/src/main/java/org/bitcoinj/core/AbstractBlockChain.java

```diff
@@ -600,7 +600,7 @@ private void connectBlock(final Block block, StoredBlock storedPrev, boolean exp
                         block.getHashAsString(), filteredTxHashList.size(), filteredTxn.size());
                 for (Sha256Hash hash : filteredTxHashList) log.debug("  matched tx {}", hash);
             }
-            if (expensiveChecks && block.getTimeSeconds() <= getMedianTimestampOfRecentBlocks(head, blockStore))
+            if (expensiveChecks && block.time().getEpochSecond() <= getMedianTimestampOfRecentBlocks(head, blockStore))
```

Comment:
Can we call a method that returns `Instant` here?

Reply:
See above answer, assuming you are talkion about `getMedianTimestampOfRecentBlocks()`.

Reply:
Ok